client/src/css/index.css

```diff
@@ -569,7 +570,12 @@ p {
 }
 
 .place a:hover { color: #aaa; }
-.place .link-text { border-color: #ddd; }
+.place .link-text {
+  border-color: #ddd;
+  letter-spacing: 0.05ex;
+  font-size: 0.96em;
```

Comment:
With increased spacing fs seemed a bit too big, esp. multilined, comparing to the regular text. Of course it is a win7 arial pov, it is just a shade of the full picture. Likely OSX has a better view :)